sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_models.py

```diff
@@ -132,6 +142,11 @@ class DirectoryProperties(DictMixin):
         conditionally.
     :ivar bool deleted: if the current directory marked as deleted
     :ivar dict metadata: Name-value pairs associated with the directory as metadata.
+    :ivar str encryption_scope:
+        A predefined encryption scope used to encrypt the data on the service. An encryption
+        scope can be created using the Management API and referenced here by name. If a default
+        encryption scope has been defined at the file system, this value will override it if the
```

Comment:
Original verbiage here replaces "file system" for "container". Is that more accurate or how it is right now fine?